src/sprites/RigidBody.lua

```diff
@@ -25,10 +24,13 @@ function RigidBody:init(entity, imageTable)
 
   self.velocity = gmt.vector2D.new(0, 0)
   self.inv_mass = 0.4
-  self.restitution = 0.4
+  self.restitution = 0
   self.static_friction = 0
   self.dynamic_friction = .12
+  self.air_friction = .14
   self.kinematic = false
+  self.maxFallSpeed = 13
+  self.maxXSpeed = 6
```

Comment:
`maxXSpeed` only seems to apply in conveyor belt logic. This naming might be misleading.